editoast/src/tests/pathmodel.json

```diff
@@ -0,0 +1,414 @@
+            49.5
+          ]
+        },
+        "sch": {
+          "type": "Point",
+          "coordinates": [
+            -0.3023,
+            49.5
+          ]
+        },
+        "name": "Mid_West_station",
+        "track": "TC1",
+        "duration": 0,
+        "position": 550,
+        "suggestion": true
+      },
+      {
+        "id": "Mid_East_station",
+        "geo": {
+          "type": "Point",
+          "coordinates": [
+            -0.22656,
+            49.5
+          ]
+        },
+        "sch": {
+          "type": "Point",
+          "coordinates": [
+            -0.22656,
+            49.5
+          ]
+        },
+        "name": "Mid_East_station",
+        "track": "TD0",
+        "duration": 0,
+        "position": 14000,
+        "suggestion": true
+      },
+      {
+        "id": null,
+        "geo": {
+          "type": "Point",
+          "coordinates": [
+            -0.173581338240107,
+            49.5
+          ]
+        },
+        "sch": {
+          "type": "Point",
+          "coordinates": [
+            -0.173581338240107,
+            49.5
+          ]
+        },
+        "name": null,
+        "track": "TD0",
+        "duration": 1,
+        "position": 24681.181806430042,
+        "suggestion": false
+      }
+    ]
+  },
+  "slopes": [
+    {
+      "gradient": 0,
+      "position": 0
+    },
+    {
+      "gradient": 0,
+      "position": 8014.715212014906
+    },
+    {
+      "gradient": 3,
+      "position": 8014.715212014906
+    },
+    {
+      "gradient": 3,
+      "position": 9014.715212014906
+    },
+    {
+      "gradient": 6,
+      "position": 9014.715212014906
+    },
+    {
+      "gradient": 6,
+      "position": 10014.715212014906
+    },
+    {
+      "gradient": 3,
+      "position": 10014.715212014906
+    },
+    {
+      "gradient": 3,
+      "position": 11014.715212014906
+    },
+    {
+      "gradient": 0,
+      "position": 11014.715212014906
+    },
+    {
+      "gradient": 0,
+      "position": 16014.715212014906
+    },
+    {
+      "gradient": -3,
+      "position": 16014.715212014906
+    },
+    {
+      "gradient": -3,
+      "position": 17014.715212014904
+    },
+    {
+      "gradient": -6,
+      "position": 17014.715212014904
+    },
+    {
+      "gradient": -6,
+      "position": 18014.715212014904
+    },
+    {
+      "gradient": -3,
+      "position": 18014.715212014904
+    },
+    {
+      "gradient": -3,
+      "position": 19014.715212014904
+    },
+    {
+      "gradient": 0,
+      "position": 19014.715212014904
+    },
+    {
+      "gradient": 0,
+      "position": 26695.897018444946
+    }
+  ],
+  "curves": [
+    {
+      "radius": 0,
+      "position": 0
+    },
+    {
+      "radius": 0,
+      "position": 26695.897018444946
+    }
+  ],
+  "geographic": "0102000020E610000004000000E090555F7A32D4BF0000000000C04840D7A3703D0AD7D3BF0000000000C048408B6CE7FBA9F1D2BF0000000000C048404EF577CDE937C6BF0000000000C04840",
```

Comment:
Is that valid GeoJSON? :o

Reply:
LOL, i don’t remember where i got that.
the file isn’t used, removed

Reply:
surprisingly enough
![Screenshot_20230621_125504](https://github.com/osrd-project/osrd/assets/24652545/2cd5961b-419e-49fc-8360-2a114928ec61)

Reply:
Well that's a smart way to reduce the length of the response that might be useful for us actually 😮